packages/frontend/core/src/components/pure/header/index.tsx

```diff
@@ -5,56 +5,62 @@ import {
 } from '@affine/component/app-sidebar';
 import { useIsTinyScreen } from '@toeverything/hooks/use-is-tiny-screen';
 import clsx from 'clsx';
-import { type Atom, useAtomValue } from 'jotai';
+import { useAtomValue } from 'jotai';
 import type { ReactNode } from 'react';
-import { forwardRef, useRef } from 'react';
+import { forwardRef, useCallback, useRef, useState } from 'react';
 
 import * as style from './style.css';
-import { WindowsAppControls } from './windows-app-controls';
 
 interface HeaderPros {
   left?: ReactNode;
   right?: ReactNode;
   center?: ReactNode;
-  mainContainerAtom: Atom<HTMLDivElement | null>;
   bottomBorder?: boolean;
 }
 
 // The Header component is used to solve the following problems
 // 1. Manage layout issues independently of page or business logic
 // 2. Dynamic centered middle element (relative to the main-container), when the middle element is detected to collide with the two elements, the line wrapping process is performed
 export const Header = forwardRef<HTMLDivElement, HeaderPros>(function Header(
```

Comment:
Seems forwardRef is unnecessary, I didn't find anywhere using it.

Reply:
I haven't changed this layout component yet, but I can remove the forward ref